conf/test_star_salmon.config

```
@@ -19,12 +19,11 @@ params {
     gene_counts = 'testdata/QDESQ/star_salmon/'
```

Path ist not complete: 
```suggestion
    gene_counts = 'https://raw.githubusercontent.com/qbic-pipelines/rnadeseq/dev/testdata/testdata/QDESQ/star_salmon/'
```

---

If this is not possible because it's an entire folder you could add a section in the usage on how to run the test profiles. It's possible to download an entire folder with the following command to run the test at least locally: 
`curl https://codeload.github.com/qbic-pipelines/rnadeseq/tar.gz/master | tar -xz --strip=2 rnadeseq-master/testdata/QDESQ`

---

Nice command! Docs are updated